src/pow.cpp

```diff
@@ -75,10 +76,12 @@ unsigned int CalculateNextWorkRequired(arith_uint256 bnAvg, int64_t nLastBlockTi
 unsigned int BitcoinGetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHeader *pblock, const Consensus::Params& params)
 {
     assert(pindexLast != nullptr);
-    unsigned int nProofOfWorkLimit = UintToArith256(params.PowLimit(false)).GetCompact();
-
-    // Only change once per difficulty adjustment interval
-    if ((pindexLast->nHeight+1) % params.DifficultyAdjustmentInterval() != 0)
+    unsigned int nProofOfWorkLimit = UintToArith256(params.powLimit).GetCompact();
```

Comment:
The change of this function should be reverted. I just changed it back to the same as Bitcoin Core.